### ISSUE TITLE
Backend: Bots Listing API fails with 500 server Error

### ISSUE THREAD

**Issue by Sachin-chaurasiya** [open-metadata/OpenMetadata#4715]
**Affected module**
Does it impact the UI, backend or Ingestion Framework?
Backend

**Describe the bug**
Request : http://localhost:8585/api/v1/bots?limit=10
Response 
```
{
  "responseMessage": "An exception with message [java.sql.SQLSyntaxErrorException: Table 'openmetadata_db.bots_entity' doesn't exist [statement:\"/* BotsDAO.listCount */ SELECT count(*) FROM bots_entity WHERE deleted = FALSE\", arguments:{positional:{}, named:{}, finder:[]}]] was thrown while processing request."
}
```

**Expected behavior**
Should return a list of bots users.

**Version:**
 - Python version:
 - OpenMetadata version: [e.g. 0.8]
 - OpenMetadata Ingestion package version: [e.g. `openmetadata-ingestion[docker]==XYZ`]

**Additional context**
SQL Query should be `openmetadata_db.bot_entity` instead of `openmetadata_db.bots_entity`.
<img width="211" alt="image" src="https://user-images.githubusercontent.com/59080942/166748923-26114167-69d7-4381-9f78-78885e872d8b.png">

**Comment by harshach** [comment]
@Sachin-chaurasiya we do not have bots entities as separation right now. What we need is to list and filter the users with isBot set to true

**Comment by sureshms** [comment]
@Sachin-chaurasiya this is fixed after addressing #4749. If not, reopen this issue.
